dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp

```diff
@@ -0,0 +1,19 @@
+(librepcb_device 41c899fa-89b1-4d23-93a1-e03f6f5e88eb
+ (name "SGP30")
+ (description "Gas sensor for indoor air quality applications.")
+ (keywords "gas,voc,tvoc,co2eq,sensor")
```

Comment:
Maybe add 'sensirion' as a keyword?

Reply:
Not sure if we should add the manufacturer of every component to the keywords, but it doesn't hurt.